### ISSUE TITLE
V 1.0.15 a fix for delete email needed

### ISSUE THREAD

**Issue by PeopleInside** [uvdesk/community-skeleton#447]
# Bug report

On the version V 1.0.15 the deletion function for email when ticket is created has been added.
There are an important issue for me:

The system delete permanently emails after ticket is created, this should be not happen.
Email must be moved to the trash folder because we need track received email, also sometimes need check email headers.

Delete should not delete permanently email but move in the trash as OsTicket do.
Delete permanently an email can be dangerous as will be no track of the received email so operator or admins with email access cannot verify email headers that has generated the ticket.

You may convert delete to a move trash function.
In OsTicket the option is delete email (move into trash) or move into folder with the possibility to specify the folder:

![SharedScreenshot](https://user-images.githubusercontent.com/5006150/130407051-c16a4f7d-506d-443f-907b-385b914878c0.jpg)

I never tested if select delete on OsTicket will move in default trash folder but I think so, permanent deletion of an email that generated ticket is dangerous as you remove the track to check if ticket has been created by a real email address and not fake one also can be useful for diagnosis.

**Comment by papnoisanjeev** [comment]
@PeopleInside 

Thanks for your suggestion for this update.
If possible we will defiantly try to move into trash folder instead of permanent delete email from inbox after fetch.

**Comment by PeopleInside** [comment]
> 
> 
> @PeopleInside
> 
> Thanks for your suggestion for this update.
> If possible we will defiantly try to move into trash folder instead of permanent delete email from inbox after fetch.

Yes please, until this is not done and supported I will be not able to use delete function as I cannot loose forever email that open a ticket... so I will really wait for this fix. Thanks!

PS: If also you are able to implement another check box as in the screenshot to decide the folder where move emails as in OsTicket will be also amazing.

The final scope is let admin decide if delete (move to trash) or move to a different folder, for example to a specific Gmail label.
See the screenshot above in the previous message.

Thank you!
